.travis.yml

```diff
@@ -12,6 +12,8 @@ notifications:
 # Install packages
 install:
   - pip install -r requirements.txt
+  - pip install -q mne
```

Comment:
As a small point, these lines should be removed. Travis doesn't run the examples, and doesn't need these dependencies. If we add these examples, these requirements should be listed in `requirements-docs.txt`